### PR TITLE
Changed the id constraint to fhirId and also changed the deprecated I…

### PR DESCRIPTION
**Pull request by ayku686**
…d constraint of Fhir package to FhirId. In jitsi package changed the type of activity to activity?

upgraded jisti_meet package to newer version jisti_meet_fix

All the packages needed to be upgraded since installation was not finishing. Many developers were facing issue during installation at their local machine

### CODE REVIEW

File: lib/domain/models/person.dart [hunk 1]
```diff
@@ -75,7 +75,7 @@ class PersonAddress {
 
    Address get address {
     return Address(
-      id: uuid,
+      fhirId: uuid,
```
Comment (angshu) [inline lib/domain/models/person.dart:78]: why should the variable name be changed from id to fhirId?

File: lib/screens/models/patient_view.dart [hunk 2]
```diff
@@ -13,7 +13,7 @@ class PatientModel {
   }
 
   String get uuid {
-    return _patient.id?.value ?? '';
+    return _patient.fhirId?.toString() ?? '';
```
Comment (angshu) [inline lib/screens/models/patient_view.dart:16]: similar question ..
patient.id? is more understandable from _patient.fhirId

**Comment by angshu** [comment]
I think only jitsi_meet_fix is required at this point. Others we can migrate going forward

File: lib/screens/models/patient_view.dart (hunk 2 above)
Comment (ayku686) [reply lib/screens/models/patient_view.dart:16]: The variable fhirId is declared everywhere of type string? 
Using id? gives error that id is not declared

File: lib/domain/models/omrs_patient.dart [hunk 3]
```diff
@@ -17,7 +18,7 @@ class OmrsPatient {
 
   fhir.Patient toFhir() {
     return fhir.Patient(
-      id: fhir.Id(uuid),
+      fhirId: FhirId(uuid).toString(),
```
Comment (angshu) [inline lib/domain/models/omrs_patient.dart:21]: also you don't need to case uuid to fhirId and back to string again ..
just fhirId: uuid should suffice

**Comment by ayku686** [comment]
Dear sir
Fhir no longer supports the Id class and in the newest version on it Id
class is replaced by FhirId

**Pull request closed without merging**